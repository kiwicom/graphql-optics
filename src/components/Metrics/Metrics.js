import React from 'react'

import vars from '../../styles/vars'

const style = {
  fontFamily: vars.fontFamily,
}

export default ({ trace }) => (
  <dl style={style}>
    <dt>Duration</dt>
    <dd>{parseInt(trace._source.metrics.duration / 10 ** 6)} ms</dd>
  </dl>
)
