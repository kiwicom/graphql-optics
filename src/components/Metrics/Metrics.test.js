import React from 'react'
import Metrics from './Metrics'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const data = {
    _source: {
      metrics: { duration: 1 },
    },
  }
  const tree = renderer.create(<Metrics trace={data} />).toJSON()
  expect(tree).toMatchSnapshot()
})
