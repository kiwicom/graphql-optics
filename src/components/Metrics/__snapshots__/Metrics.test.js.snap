// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<dl
  style={
    Object {
      "fontFamily": "sans-serif",
    }
  }
>
  <dt>
    Duration
  </dt>
  <dd>
    0
     ms
  </dd>
</dl>
`;
